hstsparser.py

```diff
@@ -123,78 +123,90 @@ def date_round(date: datetime.datetime) -> datetime.datetime:
 group.add_argument("--firefox", action="store_true", help="Process a Firefox database")
 group.add_argument("--chrome", action="store_true", help="Process a Chrome database")
 
-args = parser.parse_args()
-
-dirtydb = args.database_file.read()
-database = []
-
-if args.firefox:
-    dirtydb = dirtydb.split("\n")
-    for i in dirtydb:
-        if i != "":
-            record = re.split(r"\t+", i)
-            record.append(record[0][-4:])
-            record[0] = re.search(r"^([^:\^]+)", record[0]).group(0)
-            record[2] = serial_date_to_string(record[2])
-            cleaned = record[3].split(",")
-            try:
-                record[3] = datetime.datetime.fromtimestamp(int(cleaned[0]) / 1000)
-            except (OSError, ValueError):
-                record[3] = datetime.datetime.fromtimestamp(32503680000)  # 3000-01-01 00:00:00
-            if int(cleaned[2]):
-                record.append("Yes")
-            else:
-                record.append("No")
-            if args.csv_file:
-                record[3] = date_round(record[3])
-            database.append(record)
-    print_if_no_args(
-        database,
-        ["URL", "Visits", "Last Accessed", "Expiry", "Type", "Include Subdomains"],
-    )
 
-if args.chrome:
-    dirtydb = json.loads(dirtydb)
-    for i in dirtydb:
-        current = dirtydb[i]
-        if "expect_ct" not in current:
-            if bool(current["sts_include_subdomains"]):
-                subdomains = "Yes"
-            else:
-                subdomains = "No"
-            record = [
-                i,
-                datetime.datetime.fromtimestamp(current["expiry"]),
-                subdomains,
-                datetime.datetime.fromtimestamp(current["sts_observed"]),
-            ]
-            if args.csv_file:
-                record[1] = date_round(record[1])
-                record[3] = date_round(record[3])
-            database.append(record)
-    if args.wordlist_file:
-        wordlist = args.wordlist_file.read().splitlines()
-        rainbow = []
-        for i in wordlist:
-            rainbow.append(convert_domain(i))
-        for i in database:
-            for j in range(0, len(rainbow)):
-                if i[0] == rainbow[j].decode("utf-8"):
-                    i.append(wordlist[j])
-            if len(i) == 4:
-                i.append("")
-        print_if_no_args(
-            database,
-            [
-                "Base64 URL Hash",
-                "Expiry",
-                "Include Subdomains",
-                "Last Observed",
-                "Cracked Hash",
-            ],
-        )
-    else:
+def main() -> None:
+    """
+    Entry point for command line alias.
```

Comment:
Not sure if it applies to this project, but [Flake8 D200](http://www.pydocstyle.org/en/5.0.2/error_codes.html) states that a one-line docstring should fit on one line...

Reply:
So it does. I'll move them all to one line then, surprised Flake8 didn't catch that...